app/routes/admin/email/EmailUserRoute.js

```diff
@@ -19,8 +20,8 @@ const mapStateToProps = (state, { routeParams }) => {
     initialValues: {
       ...emailUser,
       user: {
-        label: emailUser.user.fullName || '',
-        value: emailUser.user.id || ''
+        label: get(emailUser, 'user.fullName', ''),
+        value: get(emailUser, 'user.id', '')
```

Comment:
I agree that lodash `get` is nicer than `||`, nice. It is yet another inconsistency in the project, though:p

Reply:
Yeah. We should do this in a more generic way, so all mappings are equal.

The reason I did this was because the new version of flow didn't like the old version.. :cry: